### ISSUE TITLE
issue with the training

### ISSUE THREAD

**Issue by juan9999** [vicgalle/stable-diffusion-aesthetic-gradients#17]
I installed the an extension version of your code via extensions in 1111, https://github.com/AUTOMATIC1111/stable-diffusion-webui-aesthetic-gradients

I assumed it's mostly your code, so you may be able to help me.

I resized 10 images to  512x512, put them in a folder (I know it finds them as I mistyped the path and got error messages), I then create the embedding and get Done generating embedding message and the embedding is placed in the correct directory. It's done generating super quick but it only generates a 4k .pt file. Tried it with two different training sets of 10 images.

On the txt2img screen, I select a 1.5 variation that I trained with a face, then under Clip Aesthetic I select the aesthetic that I trained, and have varied the steps from 10-max, and changed the weight from .1 to 1 but nothing happens. 

Any ideas?

**Comment by zrichz** [comment]
The .PT file generated will always be just 4kb, since it is just a single vector. So the actual size here is not your issue
